docs/io.cozy.files_metadata.md

```diff
@@ -256,6 +256,7 @@ to be added to a document.
 - `contentAuthor` : {string}
 - `issueDate` : {date} Issue date of the document
 - `accountNumber` : {string} Number of the related account
+- `bicNumber`: {string} Number of the related bank
```

Comment:
Should also be added to  "Additional metadata attributes". 

In fact ` Additional metadata attributes` should contains ALL the possible fields. And after that each "section" should contain only used fields.

Reply:
Its done 🙂 thx for the explanation